app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -251,9 +236,16 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
                 }
             }
         }
-    }
+        progress_bar_bookmark.visibility=View.INVISIBLE
```

Comment:
nit - this looks like it needs formatting. 
(tip: android studio has autoformatting in Code -> Reformat)